src/helperise/string_helpers/pipoto.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param {string} str - receives a string and returns the length
+ * @returns number
+ */
+function pipoto (args) {
```

Comment:
There's already an helper method ```ka``` that does this

Reply:
The existing helper function `ka` is to count number of elements in an array. This helper `pipoto` is to count the length of a string.